### PR TITLE
feat: add node run ability to builder CLI

### LINKED ISSUE #29: Add ability to run the node
Also re-export the node crate.

### PR DESCRIPTION
**Pull request by supiket**
Closes #29

### CODE REVIEW

**Comment by supiket** [comment]
Ready for review & ready to merge after https://github.com/essential-contributions/essential-node/pull/93 goes in and node version is bumped.

File: crates/builder-cli/src/main.rs [hunk 1]
```diff
@@ -183,6 +196,15 @@ fn builder_conf_from_args(args: &Args) -> anyhow::Result<builder::Config> {
     })
 }
 
+/// Construct the node's run config from the parsed args.
+fn node_run_conf_from_args(args: &Args) -> anyhow::Result<node::RunConfig> {
+    Ok(node::RunConfig {
```
Comment (freesig) [inline crates/builder-cli/src/main.rs:201]: Doesn't need to be an error

**Review by freesig — APPROVED** [review]
Nice

**Comment by supiket** [comment]
Included in #31

**Pull request closed without merging**